src/Materials/Textures/mirrorTexture.ts

```diff
@@ -171,8 +171,7 @@ export class MirrorTexture extends RenderTargetTexture {
             Matrix.ReflectionToRef(this.mirrorPlane, this._mirrorMatrix);
             this._mirrorMatrix.multiplyToRef(scene.getViewMatrix(), this._transformMatrix);
 
-            // Clone to not mark matrices as updated
-            scene.setTransformMatrix(this._transformMatrix.clone(), scene.getProjectionMatrix().clone());
+            scene.setTransformMatrix(this._transformMatrix, scene.getProjectionMatrix());

```

Comment:
Only one question on this one ??? wonder if it does not impact  ?

Reply:
Removing the clone calls has been forgotten, see https://github.com/BabylonJS/Babylon.js/pull/9591#discussion_r548036894